stable/mariadb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 2.1.3
+version: 2.1.4
```

Comment:
Current version is 2.1.10
Please rebase from master and bumped the version to 2.1.11